snuba/datasets/events.py

```diff
@@ -18,6 +20,16 @@
 from snuba.query.timeseries_extension import TimeSeriesExtension
 from snuba.util import qualified_column
 
+# TODO: This will be a property of the relationship between entity and
+# storage. Now we do not have entities so it is between dataset and
+# storage.
+event_translator = TranslationMappers(
+    subscriptables=[
+        TagMapper(None, "tags", None, "tags"),
```

Comment:
It would be a tiny change, can do it here. But I am not sure why they are inconsistent. I followed the order of the Column class, thus, since it is two of them, it is pre-translation table, pre-translation column, post-translation table, post-translation column. Would you prefer table, table, column ,column ?

Reply:
https://github.com/getsentry/snuba/blob/e48bebe4aeecaf60bdcefc1df0b37af454142dff/snuba/clickhouse/translators/snuba/mappers.py#L32-L35

The ordering is fine, but the suffixes of the actual names are not consistent between the `to_*` and `from_*`. (Specifically `column_table` pairs with `table_name` and `column_name` pairs with `col_name`.)

Reply:
yes makes sense. I thought you were talking about theorder. I can change it here.